src/daemon/configfile.c

```diff
@@ -287,6 +287,8 @@ static int dispatch_loadplugin (const oconfig_item_t *ci)
 	/* default to the global interval set before loading this plugin */
 	memset (&ctx, 0, sizeof (ctx));
 	ctx.interval = cf_get_default_interval ();
+	ctx.flush_interval = 0;
+	ctx.flush_timeout = 0;
 
 	for (i = 0; i < ci->children_num; ++i) {
```

Comment:
Can you please add the following variable here and use that instead of `ci->children[i]` below? This is how we do it almost everywhere else and this seems a good time to clean up this legacy :)

```c
oconfig_item_t *child = ci->children + i;
```